Lecture2/Code/data_types/string/string.py

```diff
@@ -0,0 +1,14 @@
+sample_string = "Please be ready @ 5 pm!"
+print(sample_string)
+
+# Splicing
```

Comment:
slicing?

Reply:
Oh yes, didn't notice. Slicing seems better word here, splice means to join. It can be modified.

Reply:
once the changes are done you can only approve the pull request pr. I added a rule that there should be 2 approvers